Sources/STTextViewAppKit/STTextView+Gutter.swift

```diff
@@ -72,7 +72,49 @@ extension STTextView {
         let selectedLineTextAttributes: [NSAttributedString.Key: Any] = [
             .foregroundColor: (gutterView.selectedLineTextColor ?? gutterView.textColor).cgColor
         ]
-
+        
+        if viewportRange.isEmpty {
+            let lineNumber = 1
+            
+            var lineTextAttributes: [NSAttributedString.Key: Any] = [
+                .font: gutterView.font,
+                .foregroundColor: NSColor.secondaryLabelColor
+            ]
```

Comment:
```suggestion
            var effectiveLineTextAttributes = lineTextAttributes
```
`lineTextAttributes` is already defined above.